apps/music/js/metadata/id3v2.js

```diff
@@ -140,49 +140,63 @@ var ID3v2Metadata = (function() {
    * @param {Metadata} metadata The (partially filled-in) metadata object.
    */
   function parseFrame(header, blobview, metadata) {
-    var frameid, framesize, frameflags, frame_unsynchronized = false;
+    var frameid, framesize, frameflags, frame_unsynchronized = false,
+        data_len_indicator = false, unsupported_flags = false;
 
     switch (header.version) {
     case 2:
       frameid = blobview.readASCIIText(3);
       framesize = blobview.readUint24();
       frameflags = 0;
+      unsupported_flags = (frameflags !== 0);
       break;
     case 3:
       frameid = blobview.readASCIIText(4);
       framesize = blobview.readUnsignedInt();
       frameflags = blobview.readUnsignedShort();
+      unsupported_flags = (frameflags !== 0);
       break;
     case 4:
       frameid = blobview.readASCIIText(4);
       framesize = blobview.readID3Uint28BE();
       frameflags = blobview.readUnsignedShort();
       frame_unsynchronized = ((frameflags & 0x02) !== 0);
+      data_len_indicator = ((frameflags & 0x01) !== 0);
+      unsupported_flags = ((frameflags & 0xFC) !== 0);
```

Comment:
Since frameflags is a short, shouldn't you use 0xFFFC here?

Actually, I think it would more clearly express the mask value to do ~0x03 instead of 0xFFFC